test/NuGet.Tests.Apex/NuGet.Tests.Apex/Apex/NuGetUIProjectTestExtension.cs

```diff
@@ -62,6 +62,23 @@ public void AssertInstalledPackageNotDeprecated()
             DeprecatedPackageResult.Should().BeFalse();
         }
 
+        public void AssertPackageTransitive(string testPackageName, string transitivePackageName)
+        {
+            var transitivePackageResult = _uiproject.VerifyTopLevelAndTransitivePackageOnInstalledTab(testPackageName, transitivePackageName);
+            transitivePackageResult.Should().BeTrue();
+        }
+
+        public void AssertPackageNotTransitive(string testPackageName, string transitivePackageName)
```

Comment:
This method asserts 2 things, so if it fails, it's not obvious which one failed. 

The best approach would be to get the list inside the main test method, and then assert that the first value is top level and the 2nd is transitive. That would give you a better error message in case of a failure.

Reply:
Thanks for the suggestion, we have split this method by asserting that the first value is the top-level value and the second value is the pass-through value.